UI/window-basic-main-profiles.cpp

```diff
@@ -829,6 +832,12 @@ void OBSBasic::CheckForSimpleModeX264Fallback()
 			qsv_supported = true;
 		else if (strcmp(id, "ffmpeg_nvenc") == 0)
 			nve_supported = true;
+#ifdef ENABLE_HEVC
+		else if (strcmp(id, "jim_hevc_nvenc") == 0)
+			nve_supported_hevc = true;
```

Comment:
Don't need to test for jim_hevc_nvenc. Would match H.264 pattern.

Reply:
Done.